stl/inc/type_traits

```diff
@@ -2058,49 +2058,49 @@ struct _Is_trivially_swappable : bool_constant<_Is_trivially_swappable_v<_Ty>> {
+        using _IntTy = _STD underlying_type_t<_BITMASK>;                                        \
+        return static_cast<_BITMASK>(static_cast<_IntTy>(_Left) & static_cast<_IntTy>(_Right)); \
+    }                                                                                           \
+                                                                                                \
+    _NODISCARD constexpr _BITMASK operator|(_BITMASK _Left, _BITMASK _Right) noexcept {         \
+        using _IntTy = _STD underlying_type_t<_BITMASK>;                                        \
+        return static_cast<_BITMASK>(static_cast<_IntTy>(_Left) | static_cast<_IntTy>(_Right)); \
+    }                                                                                           \
+                                                                                                \
+    _NODISCARD constexpr _BITMASK operator^(_BITMASK _Left, _BITMASK _Right) noexcept {         \
+        using _IntTy = _STD underlying_type_t<_BITMASK>;                                        \
+        return static_cast<_BITMASK>(static_cast<_IntTy>(_Left) ^ static_cast<_IntTy>(_Right)); \
+    }                                                                                           \
+                                                                                                \
+    constexpr _BITMASK& operator&=(_BITMASK& _Left, _BITMASK _Right) noexcept {                 \
+        return _Left = _Left & _Right;                                                          \
+    }                                                                                           \
+                                                                                                \
+    constexpr _BITMASK& operator|=(_BITMASK& _Left, _BITMASK _Right) noexcept {                 \
+        return _Left = _Left | _Right;                                                          \
+    }                                                                                           \
+                                                                                                \
+    constexpr _BITMASK& operator^=(_BITMASK& _Left, _BITMASK _Right) noexcept {                 \
+        return _Left = _Left ^ _Right;                                                          \
+    }                                                                                           \
+                                                                                                \
+    _NODISCARD constexpr _BITMASK operator~(_BITMASK _Left) noexcept {                          \
+        using _IntTy = _STD underlying_type_t<_BITMASK>;                                        \
+        return static_cast<_BITMASK>(~static_cast<_IntTy>(_Left));                              \
     }
 
+template <class _Bitmask>
+_NODISCARD constexpr bool _Bitmask_includes(_Bitmask _Left, _Bitmask _Elements) noexcept {

```

Comment:
Should we rename this `_Bitmask_includes_any`/`_some` while we're here? My intuition for what this name means is completely wrong.

Reply:
Yes please, this confused me so much